pasos.md

```diff
@@ -0,0 +1,9 @@
+1- comando git clone con ruta wiki= git clone https://github.com/gustavonecore/wiki.wiki.git
+2.- crear un branch en mi caso feature/1= git checkout -b feature/1 master
+3.-guardar cambios en la rama localmente con el comit=git commit -m "commit wiki"
```

Comment:
Capitaliza cada línea al inicio.

Agrega el formato del comment, dónde debe ir el número del issue.

Save changes inf the local branch with `git commit -m "#{ISSUE_NUMBER} :: commit wiki"`